submissions/anthoniaAdollo/HTML/myproductpage.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Nurture-All About You!</title>
+</head>    
+<body>
+    <nav>
+    <a href="#">Home</a>
+    <a href="#"> Product Categories</a>
+    <a href="#">Mommy&Me Blog</a>
+    <a href="#">Contact Us</a>
+
+    </nav>
+
```

Comment:
Please format the whole page properly before we proceed

Reply:
ok, thank you for the feedback. I'll do so now. please hold

Reply:
Changes have been made, as directed. Please review.